tendrl/node_agent/manager/manager.py

```diff
@@ -227,6 +230,35 @@ def on_pull(self, raw_data):
                     node_id=raw_data["node_id"],
                 )
             )
+        if "disks" in raw_data:
+            LOG.info("on_pull, Updating disks")
+            try:
+                self.etcd_client.delete(
```

Comment:
The prob is if all the disk are removed then raw disk will come empty, at that time all disk should be deleted in etcd, so i feel this implementation works  fine